types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts

```diff
@@ -27,7 +27,11 @@ configuration = {
                     loader: MiniCssExtractPlugin.loader,
                     options: {
                         publicPath: '/',
-                    },
+                        esModule: true,
+                        modules: {
+                            namedExport: true,
+                        }
+                    } as MiniCssExtractPlugin.LoaderOptions,
```

Comment:
you're getting `ERROR: 28:30 no-object-literal-type-assertion` with this one,
either rewrite to use `const ...:LoaderOptions` or add exclusion annotation if acceptable.
THx!

Reply:
Thanks for showing me! I pushed a new commit to address this. Can you please take a look?